### DISCUSSION TITLE
data importer resets 3rd party auth

### DISCUSSION THREAD

**Discussion by kesot** [firefly-iii/firefly-iii#6692] [Firefly III]
Hi, I tried to use FIDI to import transactions from nordigen, all works fine except that it requires to authorize it every time trough my bank auth. The access key for nordigen is stored in .fidi.env file (I use docker) and I authorized nordigen for my bank through their UI, so why is it necessary to go through auth each time I do import? Is it a miss on my side in some config value or is it by design?

**Comment by JC5** [comment]
This depends on your bank, in conjunction with Nordigen. Perhaps support@nordigen.com can help?
